src/blocks/cpu.rs

```diff
@@ -170,10 +170,18 @@ impl Block for Cpu {
             barchart.push(BOXCHARS[(7.5 * utilization) as usize]);
         }
 
+        let boost = match boost_status() {
+            Some(true) => "⏼",
+            Some(false) => "◌",
```

Comment:
I think it would make sense to add those icons into icon sets.

Reply:
It was a little more complicated :) In all sets I add the same icons, they work in all fonts.

Reply:
> In all sets I add the same icons, they work in all fonts

Are you sure though? Maybe it would be better to use different icons? You can try searching for "power" or "toggle" in https://www.nerdfonts.com/cheat-sheet

Reply:
Sorry, busy week... I will try it later